platform/osx/export/export.cpp

```diff
@@ -41,6 +41,7 @@
 #include "editor/editor_export.h"
 #include "editor/editor_node.h"
 #include "editor/editor_settings.h"
+#include "modules/modules_enabled.gen.h"
```

Comment:
That won't work on `3.x` which predates this refactoring:
```
platform/osx/export/export.cpp:44:10: fatal error: 'modules/modules_enabled.gen.h' file not found
#include "modules/modules_enabled.gen.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

But you can instead rely on `if (Engine::get_singleton()->has_singleton("GodotSharp"))` to check if there's C# support.